src/write.jl

```diff
@@ -39,33 +49,29 @@ function set_value(ptrentry, tagv)
     elseif entry.format == LibExif.EXIF_FORMAT_LONG
         LibExif.exif_set_long(entry.data, FILE_BYTE_ORDER, parse(Cuint, tagv))
     elseif entry.format == LibExif.EXIF_FORMAT_RATIONAL
-        if entry.tag in [LibExif.EXIF_TAG_FNUMBER, LibExif.EXIF_TAG_APERTURE_VALUE, LibExif.EXIF_TAG_MAX_APERTURE_VALUE]    
+        if entry.tag in (LibExif.EXIF_TAG_FNUMBER, LibExif.EXIF_TAG_APERTURE_VALUE, LibExif.EXIF_TAG_MAX_APERTURE_VALUE)    
```

Comment:
It's better to use `Tuple`s instead of `Vector`s here (size inferred at compile time).